src/collective/jsonmigrator/blueprints/source_remote.py

```diff
@@ -21,6 +20,12 @@
 _marker = object()
 MEMOIZE_PROPNAME = "_memojito_"
 
+if six.PY2:
```

Comment:
I think you can use the test below.

Reply:
I suggest https://github.com/plone/Products.CMFPlone/blob/2f02e0543d430ecfdbbb3889b440601938cf65f8/Products/CMFPlone/tests/testCookieAuth.py#L9-L12

Reply:
> We can even remove the six dependency.

Oh no. it is used elsewhere. :smile: